api/graph/src/main/java/quarks/graph/Connector.java

```diff
@@ -115,5 +115,27 @@ Licensed to the Apache Software Foundation (ASF) under one
      * 
      * @return set of tag values.
      */
-    Set<String> getTags(); 
+    Set<String> getTags();
+    
+    /**
+     * Set the alias for the connector.
+     * <p>
+     * The alias must be unique within the topology.
+     * The alias may be used in various contexts:
+     * <ul>
+     * <li>Runtime control services for the connector are registered with this alias.</li>
```

Comment:
I'm a bit confused by "Runtime control services for the connector".  At the high level, a control service provides an interface for controlling the stream and is registered with the TStream's alias.